docs/monitoring.md

```diff
@@ -35,18 +35,24 @@ Kubernetes-based environment:
     * set `pmm.enabled=true`
     * set the `pmm.serverHost` key to your PMM Server hostname or IP address
         (it should be resolvable and reachable from within your cluster)
-    * check that  the `serverUser` key contains your PMM Server user name
-        (`admin` by default),
-    * make sure the `pmmserver` key in the
-        [deploy/pmm-secret.yaml](https://github.com/percona/percona-postgresql-operator/blob/main/deploy/pmm-secret.yaml)
-        secrets file contains the password specified for the PMM Server during its
-        installation.
-
-    Apply changes with the `kubectl apply -f deploy/pmm-secret.yaml` command.
-
+    * authorize PMM Client within PMM Server: [acquire the API Key from your PMM Server](https://docs.percona.com/percona-monitoring-and-management/details/api.html#api-keys-and-authentication) and set ``PMM_SERVER_KEY`` in the [deploy/secrets.yaml](https://github.com/percona/percona-server-mongodb-operator/blob/main/deploy/secrets.yaml) secrets file to this obtained API Key value. Keep in mind that you need an API Key with the "Admin" role. The API Key won't be rotated automatically.
+    
+    !!! note
+
+        Alternatively, you can query your PMM Server installation for the API
+        Key using `curl` and `jq` utils as follows:
+        
+        ``` {.bash data-prompt="$" }
+        $ API_KEY=$(curl --insecure -X POST -H "Content-Type: application/json" -d '{"name":"operator", "role": "Admin"}' "https://<login>:<password>@<server_host>/graph/api/auth/keys" | jq .key)
+        ```
+        
```

Comment:
I believe we also should add the command to create the secret 
`kubectl create secret generic cluster1-pmm-secret --from-literal=PMM_SERVER_KEY="eyJrIjoickJrS3YxZjJGdGJUZzAycVJHOWk2dHlhSFFjb0lIWTAiLCJuIjoib3BlcmF0b3IiLCJpZCI6MX0="`
something like this